src/bluecoins_cli/adb.py

```diff
@@ -44,8 +44,8 @@ def pull_db(self, db: str) -> None:
             check=True,
         )
 
-    def cli_command_run(self, cli_command: str, db: str) -> None:
-        subprocess.run(f'bluecoins-cli {db}.fydb {cli_command}', shell=True, check=True)
+    def cli_command_run(self, cli_command: str, db: str, keys: str) -> None:
```

Comment:
I suggest renaming keys -> options/arguments.

See [argparse tutorial](https://docs.python.org/3/howto/argparse.html) to learn the difference.